view/frontend/templates/script/sentry.phtml

```diff
@@ -40,7 +40,7 @@ if (typeof Sentry !== 'undefined') {
             }),
             <?php endif ?>
             <?php if ($block->useSessionReplay()): ?>
-            new Sentry.Replay({
+            Sentry.replayIntegration({
```

Comment:
Just to be sure, we don't need the `new` keyword anymore?

Reply:
yes. it is not a constructor anymore.